package.json

```diff
@@ -8,7 +8,7 @@
     "web": "expo start --web",
     "eject": "expo eject",
     "test:watch": "jest --watchAll",
-    "test": "TZ=GMT jest",
+    "test": "SET TZ=GMT && jest --runInBand --forceExit",
```

Comment:
What are you trying to do with these changes? With these changes, `yarn test` no longer works. If you want to update the test (unfortunately it's needed at the moment) you need to run `yarn test -u`

Reply:
Okay, I will change it back to how it was. Thanks for informing.